styleguide/_themes/derek/scss/components/_tabs.scss

```diff
@@ -71,7 +71,7 @@
 }
 
 @media only screen and (max-width : 992px) {
-  .tab-content{
+  #myTab .tab-content{
```

Comment:
In addition to naming I am more concerned with selector specificity. Was this styling setting not working without the id prefixed into the selector?

Sorry if we are splitting hairs on this. I think small details like this are teachable moments for us to determine really good solutions to this stuff instead of the normal CSS/selector onslaught and expansion.